### PR TITLE
kubetail: update to 1.6.6

### PR DESCRIPTION
**Pull request by ndowens**

### CODE REVIEW

**Comment by maxice8** [comment]
@steinex

File: srcpkgs/kubetail/template [hunk 1]
```diff
@@ -1,14 +1,14 @@
 # Template file for 'kubetail'
 pkgname=kubetail
-version=1.6.5
+version=1.6.6
 revision=1
-depends="kubectl"
+depends="bash kubectl"
```
Comment (steinex) [inline srcpkgs/kubetail/template]: base-system depends on bash, I wouldn't add it as a dependency here.

Comment (steinex) [reply srcpkgs/kubetail/template]: fine then

**Comment by steinex** [comment]
OK

**Pull request merged**